src/api/json/catalog.json

```diff
@@ -4185,6 +4185,15 @@
       "description": "Uplift configuration file",
       "fileMatch": [".uplift.yml", ".uplift.yaml", "uplift.yml", "uplift.yaml"],
       "url": "https://upliftci.dev/static/schema.json"
+    },
+    {
+      "name": "QueryFirst config file",
+      "description": "Config options for QueryFirst, SQL wrapper generator.",
+      "fileMatch": ["qfconfig.json"],
+      "url": "https://queryfirst.net",
```

Comment:
Change to
"url": "https://json.schemastore.org/qfconfig.json",

Reply:
changed